builder/bundles.go

```diff
@@ -347,6 +365,14 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 				// Add packages to bundle's AllPackages
 				for _, pkg := range pkgs {
 					name := pkg.name + "-" + pkg.version + "." + pkg.arch + ".rpm"
+					if pkg.repo == "local" {
+						if _, err := os.Stat(filepath.Join(localPath, name)); os.IsNotExist(err) {
+							name = queryRpmName(packagerCmd, pkg.name)
+							if name == "" {
```

Comment:
I don't think we should check if its empty and revert back the hard-coded name. 
Since the first dnf query has already found the package in the local repo, there should not be a reason why the output of queryRpmName() is empty. If it happens to be empty for some weird reason, let it error out later in the code.